fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalUnion.java

```diff
@@ -196,6 +196,11 @@ public void computeUniform(FunctionalDependencies.Builder fdBuilder) {
         // don't propagate uniform slots
     }
 
+    @Override
+    public void computeEqualSet(FunctionalDependencies.Builder fdBuilder) {
+        // don't generate any equal pair
```

Comment:
why?

Reply:
the child of the union can break the equal relation